covalent/_serialize/common.py

```diff
@@ -0,0 +1,99 @@
+# Copyright 2021 Agnostiq Inc.
+#
+# This file is part of Covalent.
+#
+# Licensed under the GNU Affero General Public License 3.0 (the "License").
+# A copy of the License may be obtained with this software package or at
+#
+#      https://www.gnu.org/licenses/agpl-3.0.en.html
+#
+# Use of this file is prohibited except in compliance with the License. Any
+# modifications or derivative works of this file must retain this copyright
+# notice, and modified files must contain a notice indicating that they have
+# been altered from the originals.
+#
+# Covalent is distributed in the hope that it will be useful, but WITHOUT
+# ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+# FITNESS FOR A PARTICULAR PURPOSE. See the License for more details.
+#
+# Relief from the License may be granted by purchasing a commercial license.
+
```

Comment:
Can you add a one-line comment describing what this module does? Like:

"""Utils functions for electron, lattice etc. serialization."""

Reply:
@cjao @AlejandroEsquivel If you could weigh in on this as well - Would it be beneficial to have these functions grouped under some class? It's a pattern that we've used before for better organization purposes.

Reply:
Also the entire module needs docstrings.